app/models/legal_aid_application.rb

```diff
@@ -276,12 +276,12 @@ def offline_savings_value
 
   def online_savings_accounts_balance
     accounts = applicant.bank_accounts.reject { |c| c.account_type == 'TRANSACTION' }&.map(&:balance)
-    accounts.sum.to_s
+    accounts.present? ? accounts.sum.to_s : nil
   end
 
   def online_current_accounts_balance
     accounts = applicant.bank_accounts.reject { |c| c.account_type == 'SAVINGS' }&.map(&:balance)
-    accounts.sum.to_s
+    accounts.present? ? accounts.sum.to_s : nil
   end
```

Comment:
Are there definitely only two types of savings accounts, SAVINGS and TRANSACTIONS?  As things stand, if there were a third type, they would be included in both `online_savings_account_balance` and `online_current_account_balance`.

I think a better solution would be to have the `online_current_account_balance` use `#select` TRANSACTION to get all the current accounts, and leave the `online_savings_account_balance` to `#reject` TRANSACTION.  That way, you make sure you include all non-TRANSACTION account types in the savings account balance and there is no double counting.

Reply:
According to the TrueLayer docs https://docs.truelayer.com/#retrieve-an-account yes, unless we expect applicants to hev BUSINESS_TRANSACTION or BUSINESS_SAVINGS accounts :)

Reply:
Possibly could be adapted to 
```ruby
c.account_type.include?('SAVINGS')
```